Jenkinsfile

```diff
@@ -30,7 +30,7 @@ node {
         stage 'Archive results'
         
         stage 'ATH'
-        sh "cd acceptance-tests && ./run.sh -a=../blueocean/ --no-selenium --settings='-s ../settings.xml''"
+        sh "cd acceptance-tests && ./run.sh -a=../blueocean/ --no-selenium --settings='-s ../settings.xml'"
```

Comment:
perhps this should be expanded to be the full path so it is the same whereever it runs